tests/integration/mci/test_change_answers.py

```diff
@@ -47,3 +49,54 @@ def happy_path(self, eq_id, form_type_id):
         self.assertInPage('>Your responses<')
         self.assertInPage('Please check carefully before submission')
         self.assertInPage('>Submit answers<')
+
+        self.get('/dump/answers')
+
+        # Then I get a 200 OK response
+        self.assertStatusOK()
+
+        # And the JSON response contains the data I submitted
+        actual = json.loads(self.getResponseData())
```

Comment:
I wonder if we should move this into the base class now so we can call `self.dumpAnswers()` which results the parsed json dict so move lines 53 through lines 59 into the IntegrationTestCase class and just call that here

`actual = self.dumpAnswers()`